.infra/Pulumi.adhoc.yaml

```diff
@@ -56,6 +56,8 @@ config:
     kratosOrigin: http://heimdall-kratos-public
     logLevel: info
     nodeEnv: development
+    otlpCollectorHost: jaeger-collector
```

Comment:
Don't we want to point this to GCP or does it auto do both?

Reply:
This is just for local environment. If it is in production it will use the google gcloud exporter and not the opentelemetry exporter (would of course be better of gcloud supported the opentelemetry standard 🤓)